village/src/components/SmurfForm.js

```diff
@@ -1,25 +1,32 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import axios from "axios";
 
 class SmurfForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      name: '',
-      age: '',
-      height: ''
+      name: "",
+      age: "",
+      height: ""
     };
   }
 
   addSmurf = event => {
-    event.preventDefault();
+    console.log(this.state);
     // add code to create the smurf using the api
-
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
-  }
+    axios
+      .post("http://localhost:3333/smurfs", this.state)
```

Comment:
I highly recommend against passing `this.state` as the object here. Build a new object first, using the data in state, then pass that to the post method.